server/views/pages/barcode/find-recipient.njk

```diff
@@ -9,7 +9,7 @@
 
 {% block content %}
 
-  <main class="app-container govuk-body">
+  <section class="app-container govuk-body">
```

Comment:
This is the accessibility fix I applied to all pages - basically we had nested `<main>` elements which axe objected to (only a minor objection so it didnt fail our tests)
Anyway, the reason there were nested `<main>` elements is because `layout.njk` has the parent (and arguably the correct) `<main>` element, then each page had its own. I've simply changed these to `<section>` which feels semantically correct, and axe is happy again 👍